flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/batch/sql/RowLevelUpdateTest.java

```diff
@@ -136,7 +136,7 @@ void testUpdateWithCustomColumns() {
                                 "CREATE TABLE t (a int, b string, c double) WITH"
                                         + " ("
                                         + "'connector' = 'test-update-delete', "
-                                        + "'required-columns-for-update' = 'b;c', "
+                                        + "'required-columns-for-update' = '[b, c]', "
```

Comment:
The incompatibility of existing SQLs can be a problem.
I think we need to find way that allows users to use standard YAML config file without changing the existing sql jobs.